Earthfile

```diff
@@ -79,9 +79,13 @@ BUILD_GOLANG:
     ARG CGO_ENABLED
     ARG BIN
     ARG SRC
+    COPY +version/VERSION ./
+    ARG VERSION=$(cat VERSION)
     ENV CGO_ENABLED=${CGO_ENABLED}
-
-    RUN go build -ldflags "-s -w" -o ${BIN} ./cmd/${SRC} && upx ${BIN}
+    ARG LDFLAGS="-s -w -X 'github.com/kairos-io/kairos/pkg/common.VERSION=$VERSION'"
```

Comment:
There is no need to expose this in `pkg`, so better to consume a var inside `internal`. Also use camelcase here, I think the linter will complain

Reply:
Ah, thanks for reminding me about `internal`!  I should use that, indeed!

Reply:
Done.